src/clr-angular/wizard/wizard-page.ts

```diff
@@ -193,6 +193,37 @@ export class ClrWizardPage implements OnInit {
   @Output('clrWizardPagePreviousDisabledChange')
   public previousStepDisabledChange: EventEmitter<boolean> = new EventEmitter();
 
+  /**
+   *
+   * @memberof WizardPage
+   *
+   */
+  private _hasError = false;
+
+  /**
+   * A getter that tells whether or not the page has an error.
+   *
+   * @memberof WizardPage
+   *
+   */
+  public get hasError() {
+    return this._hasError;
+  }
+
+  /**
+   * Sets whether the page has an error.
```

Comment:
We should put a note in here that says that `hasError` resolves "falsy" values. The current logic will treat a `0` or an empty string as false. And likewise will treat any "truthy" value as true.

This is fine and we do it in other places. But we should point it out here so that it won't be hard for us to track down if a user messes up and passes weird values in through `clrWizardPageHasError`